bigbone-rx/src/main/kotlin/social/bigbone/rx/RxAnnouncementMethods.kt

```diff
@@ -0,0 +1,44 @@
+package social.bigbone.rx
+
+import io.reactivex.rxjava3.core.Completable
+import io.reactivex.rxjava3.core.Single
+import social.bigbone.MastodonClient
+import social.bigbone.api.Pageable
+import social.bigbone.api.entity.Announcement
+import social.bigbone.api.method.AnnouncementMethods
+
+/**
+ * Reactive implementation of [AnnouncementMethods].
+ * Allows access to API methods with endpoints having an "api/vX/announcements" or "api/vX/announcement" prefix.
```

Comment:
All endpoints seem to use `api/v1/announcements` and none use the singular form, so I’d suggest to change this.

```suggestion
 * Allows access to API methods with endpoints having an "api/vX/announcements" prefix.
```

Reply:
@AnuraagReddy123 You marked this as resolved but the part is still there